packages/cli/lang/en.lyaml

```diff
@@ -1119,8 +1121,8 @@ en:
           errors:
             buildIdDoesNotExist: "Build {{ buildId }} does not exist for project {{ projectName }}."
             buildAlreadyDeployed: "Build {{ buildId }} is already deployed."
-        overwritePrompt:
-          message: "Are you sure you want to overwrite {{ filePath }} on HubSpot?"
+        cleanUploadPrompt:
+          message: "You are about to remove any remote files in \"{{ filePath }}\" on HubSpot account {{ accountId }} that don't exist locally. Are you sure you want to do this? y/N"
```

Comment:
Nit: remove y/N text in this string

Reply:
I think it's helpful to give an indicator what the default is

Reply:
Looks like the prompting lib already supplies the y/N so no need to have it in the string. Removed.

![image](https://github.com/HubSpot/hubspot-cli/assets/48874841/d88bb97f-2cf5-47d0-875f-e78128dbd872)

Reply:
Oh interesting! My brain must have trimmed that out while testing 😆